services/IoT/applications/libraries/python.md

```diff
@@ -604,33 +599,33 @@ The *retrieveDevices()* method can be used to retrieve all the devices of a part
 
 
 
-In order to pass the *_bookmark* or any other condition, the overloaded method must be used. The overloaded method takes the parameters in the form of dictionary as shown in the following example:
+In order to pass the *_bookmark* or any other condition, the overloaded method must be used. The overloaded method takes the parameters in the form of a dictionary, which is shown in the following example:
 
 ```
     response = apiClient.retrieveDevices("iotsample-ardunio", parameters);
 ```
 
-The previous code sample sorts the response based on device ID, and then uses the bookmark to page through the results.
+The previous code sample sorts the response based on device ID and then uses the bookmark to page through the results.
 
 ### Adding a device
 
 
-The *registerDevice()* method is used to add a device to a {{site.data.keyword.iot_short_notm}} organization. The *registerDevice()* method adds a single device to your {{site.data.keyword.iot_short_notm}} organization. The parameters that you can specify when you add a device are:
+The *registerDevice()* method is used to add a device to a {{site.data.keyword.iot_short_notm}} organization. The *registerDevice()* method adds a single device to your {{site.data.keyword.iot_short_notm}} organization. You can specify the following parameters when you add a device:
 
-- deviceTypeId: *Optional*. Assigns a device type to the device. If there is a conflict between variables defined by the device type and variables defined by deviceInfo, the device specific variables will take precedence.
+- deviceTypeId: *Optional*. Assigns a device type to the device. If there is a conflict between the variables that are defined by the device type and the variables that are defined by deviceInfo, the device-specific variables will take precedence.
 - deviceId: *Mandatory*.
 - authToken: *Optional*. If no authentication token is supplied, one will be generated and included in the response.
-- deviceInfo: *Optional*. This parameter is optional, and can contain a number of variables, including: serialNumber, manufacturer, model, deviceClass, description, firmware and hardware versions, and descriptiveLocation.
-- metadata: *Optional*. Metadata can optionally be added in the form of custom field:value string pairs. An example is given in the next code sample on this page.
+- deviceInfo: *Optional*. This parameter can contain a number of variables including serialNumber, manufacturer, model, deviceClass, description, firmware and hardware versions, and descriptiveLocation.
+- metadata: *Optional*. Metadata can be added in the form of custom field:value string pairs. An example is given in the next code sample on this page.
 - location: *Optional*. This parameter contains the longitude, latitude, elevation, accuracy, and mesauredDateTime variables.
```

Comment:
That's a typo. It's measuredDateTime in the code and elsewhere. Fixed in pull request #873.